jib-core/src/main/java/com/google/cloud/tools/jib/configuration/LayerConfiguration.java

```diff
@@ -45,20 +47,47 @@ public Builder setLabel(String label) {
     }
 
     /**
-     * Adds an entry to the layer.
+     * Adds an entry to the layer. Only adds the single source file to the exact path in the
+     * container file system.
      *
-     * <p>The source files are specified as a list instead of a set to define the order in which
-     * they are added.
+     * <p>For example, {@code addEntry(Paths.get("myfile"), Paths.get("/path/in/container"))} would
+     * add {@code myfile} to the container to be accessed at {@code /path/in/container}.
      *
-     * @param sourceFiles the source files to build from. Source files that are directories will
-     *     have all subfiles in the directory added (but not the directory itself)
-     * @param destinationOnImage Unix-style path to add the source files to in the container image
-     *     filesystem
+     * @param sourceFile the source file to add to the layer
+     * @param pathInContainer the path to add the source file to in the container file system
```

Comment:
Is it the path to correspond to the source file (that is, the actual name) or the path to the directory to add the source file to?

_The path in the container file system corresponding to the {@code sourceFile}_?